biblib/views/user_view.py

```diff
@@ -190,12 +192,22 @@ def get(self):
             user = self.helper_get_user_id()
         except KeyError:
             return err(MISSING_USERNAME_ERROR)
+        
+        try:
+            get_params = request.args
+            start = get_params.get('start', default=0, type=int)
+            rows = get_params.get('rows', type=int)
```

Comment:
Let's add both `start` and `rows` to the docstring here. Once this PR is done, you should update the API docs as well

Reply:
Added start and rows to docstring.